06-regression.Rmd

```diff
@@ -1082,7 +1082,7 @@ Any other line draw in the figure would yield a sum of squared residuals greater
 * An arbitrarily chosen line in dashed red
 * Another arbitrarily chosen line in dashed green
 
-```{r, echo=FALSE}
+```{r three-lines, fig.cap="Regression line and two others.", fig.height=3.5, echo = FALSE}
```

Comment:
```suggestion
```{r three-lines, fig.cap="Regression line and two others.", fig.height=3.5, echo=FALSE}
```